zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1869,6 +1869,78 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /resources/{resourceKey}:
+    get:
+      tags:
+        - Resource
+      summary: Get resource
+      description: |
+        Returns a deployed resource.
+        :::note
+        Currently, this endpoint only supports RPA resources.
+        :::
+      parameters:
+        - name: resourceKey
+          in: path
+          required: true
+          description: The assigned key of the resource, which acts as a unique identifier for this resource.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: The resource is successfully returned.
+          content:
+            application/json:
+              schema:
+                type: "#/components/schemas/ResourceResult"
+        "404":
+          description: A resource with the given key was not found.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
+        "500":
+          description: An internal error occurred while processing the request.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
```

Comment:
🔧 There's no test case for the 500 responses in both endpoints. However, I think such tests are missing in general (I only found one for decision definition query controller). Might be nice to add, but not mandatory apparently.

Reply:
I added tests for this case.